syz-ci/manager.go

```diff
@@ -523,6 +527,11 @@ func (mgr *Manager) createTestConfig(imageDir string, info *BuildInfo) (*mgrconf
 		return nil, err
 	}
 	mgrcfg.KernelSrc = mgr.kernelDir
+	kernelBuildSrc, err := build.BuildSourcePath(mgr.buildParams())
```

Comment:
It also needs to be done in `writeConfig()` (that actually generates the config for normal fuzzing) and in `jobs.go` (e.g. so that reports from patch testing jobs are also properly symbolized).

Reply:
Done, but I caused some checkout weirdness by merging changes from the upstream branch. I'll upload the new version in a different PR.